src/cli/cli_tcp.cpp

```diff
@@ -908,6 +944,25 @@ void TcpExample::CompleteBenchmark(void)
 }
 
 #if OPENTHREAD_CONFIG_TLS_ENABLE
+void TcpExample::PrepareTLSHandshake(void)
+{
+    int rv;
+    rv = mbedtls_ssl_set_hostname(&mSslContext, "localhost");
+    if (rv != 0)
+    {
+        OutputLine("mbedtls_ssl_set_hostname returned %d", rv);
+    }
+    rv = mbedtls_ssl_set_hs_ecjpake_password(&mSslContext, reinterpret_cast<const unsigned char *>(sEcjpakePassword),
+                                             sEcjpakePasswordLength);
+    if (rv != 0)
+    {
+        OutputLine("mbedtls_ssl_set_hs_ecjpake_password returned %d", rv);
+    }
+    mbedtls_ssl_set_bio(&mSslContext, &mEndpointAndCircularSendBuffer, otTcpMbedTlsSslSendCallback,
+                        otTcpMbedTlsSslRecvCallback, nullptr);
+    mTlsHandshakeComplete = false;
+}
+
 bool TcpExample::ContinueTLSHandshake(void)
```

Comment:
I know it is existing method, but may be we can rename this as well? `ContinueTlsHandshake()`?

Reply:
done